test/TestFunctions/TestFunctions.csproj

```diff
@@ -21,6 +21,8 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.2.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
+    <PackageReference Include="System.Net.Http" Version="4.3.4" /> 
```

Comment:
Was getting "warning as error" package vulnerability build errors. Copying over references from dev branch project

Reply:
This PR addresses that: #10662

Reply:
Rebased on your PR and was able to remove these changes, thanks